tools/topology/topology2/include/components/mixin.conf

```diff
@@ -125,4 +127,6 @@ Class.Widget."mixin" {
 	no_pm 		"true"
 	core_id	0
 	cpc 910
+	num_sink_pins	1
+	num_source_pins	3
```

Comment:
is this correct @aiChaoSONG? It's surprising that the value is not aligned with the mixout 8 sinks.

Reply:
this is also very strange to me, but the test firmware mixin widget only support three source pin.
![image](https://user-images.githubusercontent.com/52959957/186858891-ff8e00c9-1c36-4c4e-a4d5-4d266a0e94b3.png)

Reply:
Indeed I see this in firmware definitions as well. @mmaka1 any idea why there isn't a symmetry between mixin and mixout?